product_docs/docs/biganimal/release/administering_cluster/notifications.mdx

```diff
@@ -1,32 +1,44 @@
-
-| Level        | Event                                                                                            | Role                             | Subscription type    |
-|--------------|--------------------------------------------------------------------------------------------------|----------------------------------|--------------------- |
-| Organization | Payment method added                                                                             | Organization owner/admin         | Digital self-service |
-| Organization | Personal access key is expiring                                                                  | Account owner                    | All                  |
-| Organization | Machine user access key is expiring                                                              | Organization owner               | All                  |
-| Project      | Upcoming maintenance upgrade on a cluster in 24 hours                                          | Project owner/editor             | All                  |
-| Project      | Successful maintenance upgrade on a cluster                                                      | Project owner/editor             | All                  |
-| Project      | Failed maintenance upgrade on a cluster                                                          | Project owner/editor             | All                  |
-| Project      | Paused cluster will automatically reactivate in 24 hours                                         | Project owner/editor             | All                  |
-| Project      | Paused cluster was automatically reactivated                                                     | Project owner/editor             | All                  |
-| Project      | You must set up the encryption key permission for your CMK-enabled cluster                       | Project owner/editor             | All                  |
-| Project      | Key error with CMK-enabled cluster                                                               | Project owner and project editor | All                  |
-| Project      | User is invited to a project (displays only to the Project owner)                                | Project owner                    | All                  |
-| Project      | New role is assigned to you                                                                      | Account owner                    | All                  |
-| Project      | Role is unassigned from you                                                                      | Account owner                    | All                  |
-| Project      | Failed connection to third-party monitoring integration (and future non-monitoring integrations) | Project owner/editor             | All                  |
+Different types of events are sent as notifications. Users with different roles can configure these notifications. 
```

Comment:
```
Users with different roles can configure these notifications.
```

User with different roles can only configure the preference of how to receive different notifications they can be notified.